src/patternfly/components/Nav/examples/Navigation.css

```diff
@@ -1,4 +1,5 @@
-.ws-core-c-navigation:not([id*="tertiary"]):not([id*="light-mode"]) .ws-preview-html {
+.ws-core-c-navigation:not([id*="tertiary"]):not([id*="light-mode"]) .ws-preview-html,
+#ws-core-c-navigation-expandable-third-level .pf-v5-c-nav {
```

Comment:
Nice! I overlooked those selectors. Looks like there are 2 more we may need to add? https://github.com/patternfly/patternfly/commit/447630c3449372332ee1628268397f16f5ffa965

However, I don't see why we're adding those that way... all of the navs that aren't targeting `.pf-v5-c-nav`, if the viewport is shorter than the nav height, if you scroll down, [the background is missing](https://www.patternfly.org/components/navigation/html/expanded/). I wonder if we could just update this block one of a couple of ways https://github.com/patternfly/patternfly/blob/825d0af0912528722280460f17c82acf5b66c241/src/patternfly/components/Nav/examples/Navigation.css#L1-L3

Either update the selector to target all nav elements:
```
.ws-core-c-navigation:not([id*="tertiary"]):not([id*="light-mode"]) .pf-v5-c-nav
```

Or nix the `pf-v5-u-h-100` utility class on those blocks with an override like:
```
 .ws-core-c-navigation:not([id*="tertiary"]):not([id*="light-mode"]) .ws-preview-html {
   height: auto !important;
   background-color: var(--pf-v5-global--BackgroundColor--dark-300); 
 } 
```

Just an aside, I don't think we need `pf-v5-u-h-100` anymore. That shouldn't be necessary since we added the 100vh default height to `.pf-c-page`. We can also nix the `height: 100%` from all of its ancestors (`#root, #ws-router, #ws-page`)

<img width="620" alt="Screenshot 2023-09-05 at 2 33 25 PM" src="https://github.com/patternfly/patternfly/assets/35148959/00ad17ed-7808-4e80-afc4-c5c9d809df04">